test/CMakeLists.txt

```diff
@@ -32,6 +32,22 @@ else()
     message(STATUS "oneDPL: -fno-fast-math is not supported by current compiler")
 endif()
 
+# usage of Kernel names in tests
+set(TEST_EXPLICIT_KERNEL_NAMES AUTO CACHE STRING "Use explicit Kernel names in tests")
```

Comment:
This line strongly required.
Otherwise we have CMake errors like "set_property could not find CACHE variable TEST_EXPLICIT_KERNEL_NAMES"

Reply:
Will we be able to override it by calling `cmake -DTEST_EXPLICIT_KERNEL_NAMES=OFF ...`?